addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2861,6 +2861,104 @@ options.registry.topMenuColor = options.Class.extend({
     },
 });
 
+options.registry.BreadcrumbOptions = VisibilityPageOptionUpdate.extend({
+    pageOptionName: "breadcrumb_visible",
+    showOptionWidgetName: "regular_breadcrumb_visibility_opt",
+
+    async visibility(previewMode, widgetValue, params) {
+        await this._super(...arguments);
+        await this._changeVisibility(widgetValue);
+        const targetWindow = this.$target[0].ownerDocument.defaultView;
+        targetWindow.dispatchEvent(new targetWindow.Event("resize"));
```

Comment:
Why not copying the comment that came with that copy paste? Or, better, share the code somehow? Can't this be just moved in the parent class?